server/README.md

```diff
@@ -19,7 +19,119 @@ Domain: `covid19app.who.int`
 
 Domain: `staging.whocoronavirus.org`
 
-## Curl Testing
+### [`who-mh-v2-dev1`(https://console.cloud.google.com/home/dashboard?project=who-mh-v2-dev1)
```

Comment:
`who-mh2-dev1`, I think you can remove the `mh-v2` reference as you're the only one that's using the project and it will be deleted soon anyway.